public/components/visualizations/charts/lines/line.tsx

```diff
@@ -36,10 +35,8 @@ export const Line = ({ visualizations, layout, config }: any) => {
 
   const {
     data: {
-      rawVizData: {
-        data: queriedVizData,
-        jsonData,
-        metadata: { fields },
+      explorer: {
+        explorerData: { jsonData },
```

Comment:
Do we type here: `{ jsonData: fieldValueMapList }` ?

Reply:
I'll cover this later as I'm planning to change the naming and structure more in upstream level that would touch similar these again.